### DISCUSSION TITLE
wslstorestorage.blob.core.windows.net is unreachable

### DISCUSSION THREAD

**Discussion by CC007** [microsoft/WSL#7110] [General] [answered]
### Windows Build Number

20H2 (build 19042.1052)

### Distro Version

Ubuntu 18.04

### Repro Steps

1. Go to https://wslstorestorage.blob.core.windows.net/wslblob/Ubuntu_1804.2019.522.0_x64.appx (or https://aka.ms/wsl-ubuntu-1804)
2. Try to save as file
3. Wait until download is completed

Alternatively, try to download from the commandline or install through the windows store

### Expected Behavior

Download (or installation) completes in a reasonable amount of time (My internet connection is 300Mbit/s, but I'd at the very least expect 0.5Mbit/s

### Actual Behavior

Download happens at 5KB/s and often fails, requiring the download to restart, resulting in being unable to download (or install) Ubuntu 18.04 for WSL

Same happens for Ubuntu 20.04.

I also found https://wsldownload.azureedge.net/CanonicalGroupLimited.Ubuntu18.04onWindows_1804.2018.817.0_x64__79rhkp1fndgsc.Appx 
This does download slightly faster at 50KB/s, but that is still WAY below modern standards.

![image](https://user-images.githubusercontent.com/5381337/123160027-ff9dfd80-d46d-11eb-864f-831c43ab3399.png)

I checked my internet and it was working fine. Also, https://downforeveryoneorjustme.com/ and https://www.isitdownrightnow.com/ both showed it being down.

**Comment by supardi-max** [comment]
so suck, even with new cdn 4 years later still down
<img width="1347" height="803" alt="image" src="https://github.com/user-attachments/assets/544db35e-a316-4897-833e-46dfc7860ff5" />

**Comment by OneBlue** [comment]
@CC007: Do you still see the issue ? That was most likely a transient issue.

**Comment by CC007** [comment, marked as answer]
I don't have the ability to test this right now, since I'm not at home right now. I'll close it for now and if needed reopen it if I experience it again.
